### PR TITLE
Implement submodules per unique scene

### PR DESCRIPTION
**Pull request by Anthogonyst**
As per issue #8 , I created an internal module so that each scene has a generic tick method and an unique namespace. Cheers.

### CODE REVIEW

File: app/scenes/gameplay.rb [hunk 1]
```diff
@@ -1,7 +1,12 @@
 module Scene
+  def self.tick_gameplay(args)
+    GameplayScene.tick(args)
+  end
+
+module GameplayScene
```
Comment (brettchalupa) [inline app/scenes/gameplay.rb:6]: Would you mind indenting all of these modules and their associated code one level? It'll make it easier to know it's a submodule of `Scene`

**Review by brettchalupa — CHANGES_REQUESTED** [review]
Thanks for this contribution! Looking good to me. One minor request for all of the changed files for formatting.

File: app/scenes/gameplay.rb (hunk 1 above)
Comment (Anthogonyst) [reply app/scenes/gameplay.rb:6]: I did that at first, but afterwards git kind of mangled the changes a little bit in a disruptive way. So it looks like the whole file gets replaced when it's not supposed to. It breaks git blame and so on. Any suggestions how to proceed?

Comment (brettchalupa) [reply app/scenes/gameplay.rb:6]: @Anthogonyst you should be able to indent it, commit those changes, and it should be all set! Let me know if you want any support with that. Thanks!

**Pull request closed without merging**